core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/MultiCloudWorkspaceService.scala

```diff
@@ -71,6 +116,15 @@ class MultiCloudWorkspaceService(userInfo: UserInfo,
 
     val workspaceId = UUID.randomUUID
     for {
+      _ <- samDAO.userHasAction(SamResourceTypeNames.billingProject, workspaceRequest.namespace, SamBillingProjectActions.createWorkspace, userInfo).flatMap {
+        case true => Future.successful()
+        case false => Future.failed(
+          new RawlsExceptionWithErrorReport(
+            errorReport = ErrorReport(
+              StatusCodes.Forbidden,
+              s"You are not authorized to create a workspace in billing project ${workspaceRequest.toWorkspaceName.namespace}")
```

Comment:
Why `workspaceRequest.toWorkspaceName.namespace` instead of `workspaceRequest.namespace`?

Reply:
@aherbst-broad this was my only question

Reply:
@cahrens no good reason, fixed.